modules/ROOT/pages/policy-mule4-tokenization.adoc

```diff
@@ -3,26 +3,26 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-[width="100%", cols="5,15"]
+[%autowidth.spread,cols="a,a"]
 |===
 >s| Policy name | Tokenization
->s| Summary      | Transforms sensitive data into a nonsensitive equivalent, named token. The data is displayed in the same format as the original value, but its actual value is not revealed at first sight. This policy is designed to work only for Anypoint Runtime Fabric deployments.
->s| Category | Security
+>s| Summary | Transforms sensitive data into a nonsensitive equivalent, named token. 
+>s| Category  | Security
 >s| First Mule version available | v4.1.1
-.4+>.^s| Returned Status Codes |
+.3+>.^s| Returned Status Codes |
 
 [%autowidth.spread,cols="a,a"]
 !===
-.2+>.^! 400 ! A token is invalid, or the tokenization failed.
-! The provided expression is invalid or returns no result to tokenize. +
-The status code to be returned can be configured.
-! 502 ! Any problem regarding the connection with the tokenization service, such as the service is unavailable, or the service is not configured.
+.3+>.^!400 ! A token is invalid, or the tokenization failed. ! The provided expression is invalid or returns no result to tokenize. 
```

Comment:
Is the exclamation supposed to be there?

Reply:
Yes these are a part of the formatting.